bedrock/firefox/tests/test_views.py

```diff
@@ -754,6 +754,22 @@ def test_get_context_data_one_star(self):
         self.assertFalse('donate_stars_url' in ctx)
 
 
+class TestFirefoxHubView(TestCase):
```

Comment:
This passes for me locally:

```python
@override_settings(DEV=False)
@patch('bedrock.firefox.views.l10n_utils.render')
class TestFirefoxFeatures(TestCase):
    @patch.object(views, 'lang_file_is_active', lambda *x: True)
    @patch('bedrock.firefox.views.switch', Mock(return_value=True))
    def test_new_firefox_page_loads_with_active_switch(self, render_mock):
        view = views.FirefoxHubView()
        view.request = RequestFactory().get('/firefox/')
        view.request.locale = 'en-US'
        eq_(view.get_template_names(), ['firefox/firefox-quantum.html'])

    @patch.object(views, 'lang_file_is_active', lambda *x: False)
    @patch('bedrock.firefox.views.switch', Mock(return_value=False))
    def test_firefox_hub_page_loads_with_inactive_switch(self, render_mock):
        view = views.FirefoxHubView()
        view.request = RequestFactory().get('/firefox/')
        view.request.locale = 'fr'
        eq_(view.get_template_names(), ['firefox/hub/home.html'])
```

Reply:
Hmm, actually - you could probably split this test out further so the switch is tested independently of the locale activation

Reply:
Oh so you mean test that if one of the two is not True then the current page should show. Will do.